### PR TITLE
add detail to site-wide template policy doc

### PR DESCRIPTION
**Pull request by bpmct**
This PR documents the fields that can be applied to template policies, as well as the behavior of the template policy on custom workspaces and workspaces that have conflicting values.

### CODE REVIEW

File: admin/templates.md [hunk 1]
```diff
@@ -20,38 +20,45 @@ If you enable the use of workspace templates, a **template policy** allows you
 to control which fields users can set and which values can be used when they
 define their workspaces.
 
+The following policies can be applied to fields:
+
+- **read** - Workspaces cannot modify the field
+- **write** - Workspaces can overrite the field
+- **append** - Workspaces can append items to the field, lists only (e.g
+  configure.start steps)
```
Comment (Emyrk) [inline admin/templates.md:28]: It works for lists or mappings. So `labels` for example is a mapping

File: admin/templates.md [hunk 2]
```diff
@@ -20,38 +20,45 @@ If you enable the use of workspace templates, a **template policy** allows you
 to control which fields users can set and which values can be used when they
 define their workspaces.
 
+The following policies can be applied to fields:
+
+- **read** - Workspaces cannot modify the field
```
Comment (Emyrk) [inline admin/templates.md:25]: Do we want to say that omitted `policy` is `read` by default?

**Review by khorne3 — APPROVED** [review]

**Review by Emyrk — APPROVED** [review]

File: admin/templates.md [hunk 3]
```diff
@@ -14,56 +12,70 @@ located in Git repositories.
...
-to control which fields users can set and which values can be used when they
-define their workspaces.
+to control which fields users can set and which values can they can use when
+defining their workspaces.
+
+Users can apply the following policies to fields:
+
+- `read` - **Default.** Workspaces cannot modify the field
+- `write` - Workspaces can overwrite the field
+- `append` - Workspaces can append lists (e.g., configure.start steps) or
```
Comment (bpmct) [inline admin/templates.md:27]: is the comma here intended? @khorne3 `e.g,`?

Comment (khorne3) [reply admin/templates.md:27]: @bpmct yup it is!

**Pull request merged**